src/natcap/invest/urban_nature_access.py

```diff
@@ -0,0 +1,315 @@
+                    'type': 'number',
+                    'units': u.none,
+                    'about': (
+                        "1 if this landcover code represents greenspace, 0 "
+                        "if not."
+                    ),
+                }
+            },
+            'about': (
+                "A table identifying which LULC codes represent greenspace."
+            ),
+        },
+        'population_raster_path': {
+            'type': 'raster',
+            'name': 'population raster',
+            'bands': {
+                1: {'type': 'number', 'units': u.none}
+            },
+            'projected': True,
+            'about': (
+                "A raster representing the number of people who live in each "
+                "pixel. This raster must be linearly projected."
+            ),
+        },
+        'admin_unit_vector_path': {
+            'type': 'vector',
+            'name': 'administrative boundaries',
+            'geometries': spec_utils.POLYGONS,
+            'fields': {},  # TODO, complete required fields (if any)
+            'about': "",  # TODO, will know more about this when I implement.
+        },
+        'greenspace_demand': {
+            'type': 'number',
+            'name': 'greenspace demand per capita',
+            'units': u.m**2,  # defined as m² per capita
+            'expression': "value > 0",
+            'about': "",  # TODO, will know more about this when I implement.
+        },
+        'search_radius': {
+            'type': 'number',
+            'name': 'search radius',
+            'units': u.m,
+            'expression': "value > 0",
+            'about': "",  # TODO, will know more about this when I implement.
+        }
+    }
+}
+
+
+_OUTPUT_BASE_FILES = {}
+_INTERMEDIATE_BASE_FILES = {
+    'resampled_population': 'population_resampled.tif',
+}
+
+
+def execute(args):
+    """Urban Nature Access.
+
+    Args:
+        args['workspace_dir'] (string): (required) Output directory for
+            intermediate, temporary and final files.
+        args['results_suffix'] (string): (optional) String to append to any
+            output file.
+        args['n_workers'] (int): (optional) The number of worker processes to
+            use for executing the tasks of this model.  If omitted, computation
+            will take place in the current process.
+        args['lulc_raster_path'] (string): (required) A string path to a
+            GDAL-compatible land-use/land-cover raster containing integer
+            landcover codes.
+        args['lulc_attribute_table'] (string): (required) A string path to a
+            CSV with the following columns:
+
+            * ``lucode``: the integer landcover code represented.
+            * ``greenspace``: ``0`` or ``1`` indicating whether this landcover
+              code is (``1``) or is not (``0``) a greenspace pixel.
+
+        args['population_raster_path'] (string): (required) A string path to a
```

Comment:
Should the docstring here also reflect that this raster should be projected in meters?

Reply:
Added for both of the raster keys in 9ae62145